connection.go

```diff
@@ -232,6 +232,17 @@ func (sc *snowflakeConn) Prepare(query string) (driver.Stmt, error) {
 }
 
 func (sc *snowflakeConn) ExecContext(ctx context.Context, query string, args []driver.NamedValue) (driver.Result, error) {
+	qid, _ := getResumeQueryId(ctx)
+	if qid == "" {
+		return sc.execContextInternal(ctx, query, args)
+	}
+	actualQuery := "select * from table(result_scan(" + qid + "))"
```

Comment:
Interesting approach. What are the performance characteristics of doing this vs jumping straight to using the `/queries/<query_id>/result` endpoint?

Reply:
/queries/<query_id>/result is more for an in progress query, and this one is to receive result of a completed query within last 24 hours.

Reply:
An endpoint to consider would be `/monitoring/queries/<query-id>`, that denotes the status of an ongoing query as well.

Reply:
@sfc-gh-cshi We use the query ID for in-progress queries. If we need to use the result_scan query we can do it ourselves by just submitting the SQL?